docs/samples/git-setup/clustertemplates/hardwareprofiles/profile-dell-r740-v1.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: clcm.openshift.io/v1alpha1 
+kind: HardwareProfile
+metadata:
+  name: profile-dell-r740-v1
```

Comment:
I've found that using`v1`, `v2`, ... is not intuitive when dealing with a large number of HardwareProfiles.
In my tests, having the BIOS / BMC / NIC versions in the names makes it much easier to switch in between them.
For example, for this one, we could have something like `dell-r740-bios-2.22.2-bmc-173`.
Just a suggestion.

Reply:
I do the same in my setup